src/pages/Profile.js

```diff
@@ -0,0 +1,70 @@
+import { shallowEqual, useSelector } from 'react-redux';
+import Alert from 'react-bootstrap/Alert';
+import { NavLink } from 'react-router-dom';
+import style from './pages.module.css';
+
+const Profile = () => {
+  const rockets = useSelector((state) => state.rockets).filter(({ reserved }) => reserved);
+  const missions = useSelector((state) => state.missions, shallowEqual).filter(({ status }) => status === 'Active member');
+
+  return (
+    <div className={style.profilecontainer}>
+      <section className={style.profilesection}>
+        <h3>My Rockets</h3>
+        {rockets && (
+        <ul className={style.profilelist}>
+          {
+                      rockets.map(({ name, id }) => (
```

Comment:
- [ ] Render a list of all joined missions and rockets.

* This should be done using `filter();` (Currently, map is being used for the rendering, please correct this to comply with the project requirements)

Reply:
@e71az The `filter();` is used on line 7:
 `const rockets = useSelector((state) => state.rockets).filter(({ reserved }) => reserved);`